README.md

```diff
@@ -39,7 +39,7 @@ privileges or be logged in as admin.
 
 ### To Deploy a sample `NIMCache` and `NIMService` instance
 
-Follow the guides [here](https://github.com/NVIDIA/k8s-nim-operator/-/tree/main/docs?ref_type=heads) to deploy sample CR instances.
+Follow the guides [here](https://github.com/NVIDIA/k8s-nim-operator/tree/main/docs) to deploy sample CR instances.
```

Comment:
I don't say this to stop you, but I made several edits to the private version of this file.  I'm dusting it off now.

Reply:
no worries, if you have a PR to handle this already, i will close this.